client/src/pages/Board.js

```diff
@@ -22,8 +22,7 @@ const useStyles = makeStyles((theme) => ({
 
 function Board() {
   const classes = useStyles();
-  const { user } = useContext(UserContext);
-  console.log("context", user);
+  const { loggedIn, user } = useContext(UserContext);
```

Comment:
```suggestion
  const { user } = useContext(UserContext);
```
I don't think you need loggedIn for this page because of the protected route